2021-04-23T18:10:31.3327600+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T18:10:31.3372707+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T18:10:31.3390968+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T18:10:31.3405533+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T18:10:31.3419669+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T18:10:33.1718017+02:00 0HM86E98QVDA7:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T18:10:46.3886780+02:00 0HM86E98QVDA7:0000001B [INF] "Nathan Caruana" logged in with "Google" provider. (107abac5)
2021-04-23T18:38:29.7454867+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T18:38:29.7501393+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T18:38:29.7524642+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T18:38:29.7542551+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T18:38:29.7561363+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T18:38:31.2159828+02:00 0HM86EOT2GFII:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:18:15.7026511+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:18:15.7072254+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:18:15.7088813+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:18:15.7100421+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:18:15.7112181+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:18:19.4177041+02:00 0HM86FF45R2J2:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:18:42.3352932+02:00 0HM86FF45R2J2:00000013 [ERR] The view '"Create"' was not found. Searched locations: ["/Views/File/Create.cshtml", "/Views/Shared/Create.cshtml", "/Pages/Shared/Create.cshtml"] (9bb2756c)
2021-04-23T19:18:42.4129292+02:00 0HM86FF45R2J2:00000013 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: The view 'Create' was not found. The following locations were searched:
/Views/File/Create.cshtml
/Views/Shared/Create.cshtml
/Pages/Shared/Create.cshtml
   at Microsoft.AspNetCore.Mvc.ViewEngines.ViewEngineResult.EnsureSuccessful(IEnumerable`1 originalLocations)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewResultExecutor.ExecuteAsync(ActionContext context, ViewResult result)
   at Microsoft.AspNetCore.Mvc.ViewResult.ExecuteResultAsync(ActionContext context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResultFilterAsync>g__Awaited|29_0[TFilter,TFilterAsync](ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResultExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.ResultNext[TFilter,TFilterAsync](State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeResultFilters()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-04-23T19:20:58.9953140+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:20:58.9998345+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:20:59.0032213+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:20:59.0047789+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:20:59.0064024+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:21:00.4962071+02:00 0HM86FGKO2GQU:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:24:13.8975039+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:24:13.9018182+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:24:13.9033524+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:24:13.9046862+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:24:13.9069913+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:24:15.1435878+02:00 0HM86FIESUO1B:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:26:05.6971706+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:26:05.7016929+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:26:05.7032084+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:26:05.7043492+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:26:05.7056882+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:26:07.0024134+02:00 0HM86FJG4U384:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:27:22.5634884+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:27:22.5682537+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:27:22.5698559+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:27:22.5758918+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:27:22.5775163+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:27:23.5450481+02:00 0HM86FK71KAHP:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:27:52.5311697+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:27:52.5355628+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:27:52.5372313+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:27:52.5385589+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:27:52.5404011+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:27:53.7915852+02:00 0HM86FKG2AMS4:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:31:16.8773166+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:31:16.8828868+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:31:16.8848375+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:31:16.8863691+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:31:16.8887487+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:31:17.8483671+02:00 0HM86FMCRSATI:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:33:33.2388499+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:33:33.2431620+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:33:33.2446922+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:33:33.2461174+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:33:33.2473222+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:33:34.1304293+02:00 0HM86FNLGHB9T:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:33:36.9467426+02:00 0HM86FNLGHB9T:00000003 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:33:39.1378899+02:00 0HM86FNLGHB9T:00000007 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:34:09.4048318+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:34:09.4092657+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:34:09.4111471+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:34:09.4123627+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:34:09.4134744+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:34:10.5419777+02:00 0HM86FO0A0PLF:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:36:00.5904406+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:36:00.5957638+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:36:00.5996834+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:36:00.6013300+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:36:00.6032432+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:36:01.4860472+02:00 0HM86FP1D2RSJ:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:37:36.3850074+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:37:36.3892203+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:37:36.4199452+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:37:36.4213614+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:37:36.4225574+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:37:37.2912970+02:00 0HM86FPTV84FU:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:38:42.0491636+02:00 0HM86FPTV84FU:0000000F [INF] Home Index accessed (f306b1a0)
2021-04-23T19:38:48.8453209+02:00 0HM86FPTV84FU:00000017 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index')
   at System.Collections.Generic.List`1.get_Item(Int32 index)
   at WebApplication1.Areas.Identity.Pages.Account.ExternalLoginModel.OnGetCallbackAsync(String returnUrl, String remoteError) in C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1\Areas\Identity\Pages\Account\ExternalLogin.cshtml.cs:line 89
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.GenericTaskHandlerMethod.Convert[T](Object taskAsObject)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.GenericTaskHandlerMethod.Execute(Object receiver, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeHandlerMethodAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeNextPageFilterAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Rethrow(PageHandlerExecutedContext context)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-04-23T19:46:49.2668950+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:46:49.2713930+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:46:49.2733770+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:46:49.2747170+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:46:49.2815747+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:46:50.1832527+02:00 0HM86FV2NJGR2:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:48:08.5566420+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:48:08.5610796+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:48:08.5632048+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:48:08.5650908+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:48:08.5696283+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:48:09.6000261+02:00 0HM86FVQDGL9I:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:48:22.3992668+02:00 0HM86FVQDGL9I:00000009 [INF] "Nathan Caruana" logged in with "Google" provider. (107abac5)
2021-04-23T19:48:22.4155707+02:00 0HM86FVQDGL9I:0000000B [INF] Home Index accessed (f306b1a0)
2021-04-23T19:50:31.5294217+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:50:31.5343506+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:50:31.5359094+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:50:31.5377096+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:50:31.5391479+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:50:32.5078879+02:00 0HM86G14VH34F:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:51:42.4027200+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:51:42.4081446+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:51:42.4096691+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:51:42.4116934+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:51:42.4129945+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:51:43.7942178+02:00 0HM86G1Q5NGID:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:51:55.6934980+02:00 0HM86G1Q5NGID:0000000F [INF] Home Index accessed (f306b1a0)
2021-04-23T19:52:01.9460508+02:00 0HM86G1Q5NGID:00000017 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index')
   at System.Collections.Generic.List`1.get_Item(Int32 index)
   at WebApplication1.Areas.Identity.Pages.Account.ExternalLoginModel.OnGetCallbackAsync(String returnUrl, String remoteError) in C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1\Areas\Identity\Pages\Account\ExternalLogin.cshtml.cs:line 89
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.GenericTaskHandlerMethod.Convert[T](Object taskAsObject)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.GenericTaskHandlerMethod.Execute(Object receiver, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeHandlerMethodAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeNextPageFilterAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Rethrow(PageHandlerExecutedContext context)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-04-23T19:52:12.2560735+02:00 0HM86G1Q5NGID:00000021 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index')
   at System.Collections.Generic.List`1.get_Item(Int32 index)
   at WebApplication1.Areas.Identity.Pages.Account.ExternalLoginModel.OnGetCallbackAsync(String returnUrl, String remoteError) in C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1\Areas\Identity\Pages\Account\ExternalLogin.cshtml.cs:line 89
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.GenericTaskHandlerMethod.Convert[T](Object taskAsObject)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.GenericTaskHandlerMethod.Execute(Object receiver, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeHandlerMethodAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeNextPageFilterAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Rethrow(PageHandlerExecutedContext context)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-04-23T19:52:17.3842478+02:00 0HM86G1Q5NGID:00000029 [INF] "Nathan Caruana" logged in with "Google" provider. (107abac5)
2021-04-23T19:52:17.3975034+02:00 0HM86G1Q5NGID:0000002B [INF] Home Index accessed (f306b1a0)
2021-04-23T19:54:03.3271533+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2021-04-23T19:54:03.3314949+02:00  [INF] Now listening on: "http://localhost:5000" (d826f4b8)
2021-04-23T19:54:03.3332903+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2021-04-23T19:54:03.3347698+02:00  [INF] Hosting environment: "Development" (c3307c92)
2021-04-23T19:54:03.3384967+02:00  [INF] Content root path: "C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1" (b5d60022)
2021-04-23T19:54:04.5769820+02:00 0HM86G346TE9P:00000001 [INF] Home Index accessed (f306b1a0)
2021-04-23T19:54:14.2663120+02:00 0HM86G346TE9P:0000000B [INF] Home Index accessed (f306b1a0)
2021-04-23T19:54:25.3730396+02:00 0HM86G346TE9P:00000013 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index')
   at System.Collections.Generic.List`1.get_Item(Int32 index)
   at WebApplication1.Areas.Identity.Pages.Account.ExternalLoginModel.OnGetCallbackAsync(String returnUrl, String remoteError) in C:\Users\Owner\Source\Repos\SecuringApps\WebApplication1\Areas\Identity\Pages\Account\ExternalLogin.cshtml.cs:line 89
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.GenericTaskHandlerMethod.Convert[T](Object taskAsObject)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.GenericTaskHandlerMethod.Execute(Object receiver, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeHandlerMethodAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeNextPageFilterAsync()
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Rethrow(PageHandlerExecutedContext context)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
2021-04-23T19:54:49.2922267+02:00 0HM86G346TE9P:00000025 [INF] "Nathan Caruana" logged in with "Google" provider. (107abac5)
2021-04-23T19:54:49.3086995+02:00 0HM86G346TE9P:00000027 [INF] Home Index accessed (f306b1a0)
